frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -235,7 +227,6 @@ class WorkspaceAdvanced extends React.Component {
 
   handleClickBtnCloseApp = () => {
     this.setState({ isVisible: false })
-    GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.APP_CLOSED, data: {} })
```

Comment:
why do you remove this ?

Reply:
It was part of a test that I forgot to reverse, sry. Changed.